src/components/visualizations/table/TableVisualization.tsx

```diff
@@ -363,14 +363,14 @@ export class TableVisualizationClass
         this.subscribers = subscribeEvents(this.scrolled, scrollEvents);
     }
 
-    private getSortFunc(header: TableHeader, sort: ISortObj): () => void {
+    private getSortFunc(header: IMappingHeader, sort: ISortObj): () => void {
```

Comment:
consider renaming argument to mappingHeader instead of header

Reply:
not worthy - too much orcurrences of such variables + we have type